src/ApiCalls.js

```diff
@@ -0,0 +1,9 @@
+const ApiCalls = {
```

Comment:
Great call breaking this out into its own file since it's a requirement for the project spec